VehicleWebAPI/Services/VehicleDatabaseService.cs

```diff
@@ -5,50 +5,67 @@
 
 namespace VehicleWebAPI.Services
 {
-    public class VehicleDatabaseService : IVehicleDatabaseService
+  public class VehicleDatabaseService : IVehicleDatabaseService
+  {
+    private readonly DatabaseContext _db;
+
+    public VehicleDatabaseService(DatabaseContext db)
+    {
+      _db = db;
+    }
+
+    //CREATE
+    public int CreateVehicle(Vehicle vehicle)
     {
-        private readonly DatabaseContext _db;
+      if (vehicle.VIN == null
+      || vehicle.Make == null
+      || vehicle.Model == null
+      || vehicle.Year == null
+      || vehicle.Miles < 0
+      || vehicle.Color == null
+      || vehicle.SellingPrice < 0
+      || vehicle.Status < 0
+      || vehicle.UserId < 0) return 0;
+      if (_db.Vehicles.FirstOrDefault(t => t.VIN == vehicle.VIN) != null) return 0;
```

Comment:
Think the logic needs to be a bit different here, things like Year will default to 0 and not be null. So maybe year > 1900? Also need to think about other validation.

Reply:
Will be addressed in a future PR :)